simple_network_sim/network_of_populations.py

```diff
@@ -382,7 +382,9 @@ def createNextStep(progression, exposed, currState):
 
     for regionID, region in exposed.items():
         for age, exposed in region.items():
-            expose(age, exposed, nextStep[regionID])
+            suscept = nextStep[regionID][(age, SUSCEPTIBLE_STATE)]
+            modifiedExposed = suscept*(1-(1-(1/suscept))**exposed)
```

Comment:
Isn't that equivalent to `suscept * (1/suscept)**exposed` ? Or am I missing something?

Reply:
I was missing something, it's actually `(1-(1/suscept))**exposed` and not `(1-(1-(1/suscept)))*exposed`. Would it make sense to set a variable to `(1-(1/suscept))**exposed` so that's it's easier to parse visually?

Reply:
I have no objections.  I'll amend it so it's easier to read